auction-app-backend/src/main/resources/application.yml

```diff
@@ -1,8 +1,8 @@
 spring:
   datasource:
-    url: jdbc:postgresql://dpg-cp0h51q1hbls73eaktmg-a.frankfurt-postgres.render.com/auction_app_cqgi
-    username: auction_app_cqgi_user
-    password: ${DB_PASSWORD}
+    url: jdbc:postgresql://localhost:5432/auction_app
```

Comment:
This file is till here

Reply:
i had to update the access token expiry, i accidentally pushed the version where i set it to last longer so i could test out some stuff in postman